Plate_2_data/1.cellprofiler_quality_control/README.md

```diff
@@ -0,0 +1,20 @@
+# Create LoadData CSV and create maximum projection images 
+
+In this module, we create a LoadData csv and run a CellProfiler pipeline that performs maximum projection on the images and saves them into a new folder.
+This pipeline names the outputted max projected images as the last z-plane.
+In our case we have 4 z-planes (p01-p03) so our images contain the last z-plane p04.
```

Comment:
```suggestion
In our case we have 4 z-planes (p01-p03) so our max projected images are found in the last z-plane p04.
```

Is this suggested edit accurate? You may also consider adding a link to where Erin previously guided you describing this consequence.

Reply:
It is linked in the wave 3 portion of this repo but I will add it here, doesn't hurt at all since this helped a lot.

As well, I do not think that is accurate because the maximum projected images are still calculated by finding the max pixel for every well/site image, it just names the outputted image with the last z-plane value. I will correct this sentence to reflect that.

Reply:
See https://github.com/WayScience/pyroptosis_signature/pull/20/commits/0b6a5953d29cbd11eea74fc8d01ecbef570459ac for change